scripts/multinode/configure-pri.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+set -euo pipefail
+
+OC_CMD="sudo -i oc --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig"
+KUBECTL_CMD="sudo -i kubectl --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig"
+
+PRI_NODE_HOST=
+PRI_NODE_ADDR=
+SEC_NODE_HOST=
+SEC_NODE_ADDR=
+
+function usage() {
+    echo "Usage: $(basename "$0") <primary_host_name> <primary_host_ip> <secondary_host_name> <secondary_host_ip>"
+    exit 1
+}
+
+function configure_system() {
+    # Disable selinux
+    # TODO: remove once selinux is working properly again
+    sudo setenforce 0
+
+    # TODO: Edit firewall rules instead of stopping firewall
+    sudo systemctl stop firewalld
+    sudo systemctl disable firewalld
+
+    # Configure the MicroShift host name
+    sudo hostnamectl hostname "${PRI_NODE_HOST}"
+
+    # Update /etc/hosts to resolve primary and secondary host names
+    if ! grep -q "${PRI_NODE_HOST}" /etc/hosts ; then
+        echo "${PRI_NODE_ADDR} ${PRI_NODE_HOST}" | sudo tee -a /etc/hosts &>/dev/null
+    fi
+    if ! grep -q "${SEC_NODE_HOST}" /etc/hosts ; then
+        echo "${SEC_NODE_ADDR} ${SEC_NODE_HOST}" | sudo tee -a /etc/hosts &>/dev/null
+    fi
+}
+
+function configure_microshift() {
+    # Clean the current MicroShift configuration and stop the service
+    echo 1 | sudo microshift-cleanup-data --all
+
+    # Configure MicroShift networking to run in multinode mode
+    cat <<EOF | sudo tee /etc/microshift/config.yaml &>/dev/null
+apiServer:
+  subjectAltNames:
+    - "10.43.0.1"
+EOF
+}
+
+function apply_scc_config() {
```

Comment:
This is needed to run the CNCF tests, but not to run in multi-node - it should be pulled out into into CI